### PR TITLE
Open Applications for Spring 2025

### PR DESCRIPTION
**Pull request by RishikeshNK**
**Spring 2025 Application Update**

- [x] Updated the `formLink` for all three applicants
	- [x] Developer
	- [x] UX Designer
	- [x] Brand Designer
- [x] Confirmed the application opening and closing dates

### CODE REVIEW

File: src/content/apply/brand-designer.md [hunk 1]
```diff
@@ -9,14 +9,14 @@ qualities:
   - Communicates well
   - Is kind
   - Can commit to 10 hours a week
-formLink: "https://sandboxf24.fillout.com/branddesignerapplication"
+formLink: "https://sandboxsp25.fillout.com/branddesignerapplication"
```
Comment (ahweigl) [inline src/content/apply/brand-designer.md:12]: so TLDR we can only have one domain for our fillout so I made it 'sandbox' and then made the exact link for each form 'SP25<role>'

Comment (ahweigl) [reply src/content/apply/brand-designer.md:12]: https://sandboxf24.fillout.com/SP25branddesignerapplication

File: src/content/apply/brand-designer.md [hunk 2]
```diff
@@ -9,14 +9,14 @@ qualities:
...
 mailLink: "https://gmail.us3.list-manage.com/subscribe?u=3b3ae33f54203ab7a839ae529&id=c2570dd048"
```
Comment (ahweigl) [inline src/content/apply/brand-designer.md:13]: note for future I think we hit the cap on newsletter signups, this is outdated and will need to be updated at some point

File: src/content/apply/developer.md [hunk 3]
```diff
@@ -6,14 +6,14 @@ qualities:
   - Is kind
   - Takes on leadership role to help out teammates
   - Can commit to 10 hours a week
-formLink: "https://sandboxf24.fillout.com/developerapplication"
+formLink: "https://sandboxsp25.fillout.com/developerapplication"
```
Comment (ahweigl) [inline src/content/apply/developer.md:9]: https://sandbox.fillout.com/SP25developerapplication

File: src/content/apply/ux-designer.md [hunk 4]
```diff
@@ -7,14 +7,14 @@ qualities:
   - Works well in teams
   - Is kind
   - Can commit to 10 hours a week
-formLink: "https://sandboxf24.fillout.com/uxdesignerapplication"
+formLink: "https://sandboxsp25.fillout.com/uxdesignerapplication"
```
Comment (ahweigl) [inline src/content/apply/ux-designer.md:10]: https://sandboxf24.fillout.com/SP25designerapplication

**Review by ahweigl — APPROVED** [review]
pog, just need to update links

File: src/content/apply/brand-designer.md (hunk 2 above)
Comment (RishikeshNK) [reply src/content/apply/brand-designer.md:13]: website re-write :prayge:

**Pull request merged**